### DISCUSSION TITLE
关于默认头像

### DISCUSSION THREAD

**Discussion by ncwzdlsd** [walinejs/waline#1781] [Q&A] [answered]
未登录用户的默认头像支持自定义吗？
以及对于
![image](https://user-images.githubusercontent.com/117651674/216084664-eca00e07-1119-4e57-8d23-5038324e971c.png)
这样的情况，能不能更换评论所在的页面，比如从articles/null改到articles/abcd

**Comment by lizheming** [comment, marked as answer]
1. https://waline.js.org/reference/server/env.html#%E6%98%BE%E7%A4%BA 可以通过 GRAVATAR_STR 自行修改头像配置，默认使用的 libavatar 的参数配置见文档 https://wiki.libravatar.org/api/
2. 这个可以自己修改 path 参数去适配
